CHANGELOG.md

```diff
@@ -9,15 +9,17 @@ Changes since last non-beta release.
 
 _Please add entries here for your pull requests that are not yet released._
 
-### Breaking Changes
-- Remove support & testing for Webpacker 3 and 4 #1274
-- Requires separate compilations for server & client bundles (see [Webpack config](https://github.com/reactjs/react-rails/tree/master/test/dummy/config/webpack)) #1274
-- Replaces WebpackManifestContainer, which searched for assets in the webpack manifest, with SeparateServerBundleContainer, which expects a single server bundle file & does not use the webpack manifest at all. #1274
-
+## Breaking Changes
+### Removed
```

Comment:
only put what's breaking under breaking

don't have subtopics

only provide list of what updates an upgrader needs to do